src/clarity-icons/clarity-icons-element.ts

```diff
@@ -36,52 +33,49 @@ ClarityIconElement.prototype.constructor = ClarityIconElement;
 
 let generateIcon =
     function (element: any, shape: string) {
+        let cIcons = ClarityIconsApi.instance;
```

Comment:
I decided to revert that variable. It was only a convenience for three lookups below. So there was no point in pushing it out of the generateIcon function. So I removed cIcons and replaced it with the more verbose ClarityIconsApi.instance.